packages/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,312 @@
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import Tab, { TabProps } from './Tab';
+import TabTitle from './TabTitle';
+
+const listStyle = css`
+  list-style: none;
+  padding: 0px;
+  margin-top: 20px;
+  margin-bottom: 5px;
+`;
+
+const listTitle = css`
+  display: inline-block;
+  background-color: ${uiColors.white};
+  color: ${uiColors.gray.base};
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 1;
+  text-transform: capitalize;
+  padding: 3px 24px;
```

Comment:
So I found a UX problem with these tabs:
https://drive.google.com/file/d/1lb-aZYBgtQJvQXGi6uuHPxia-XlX4X7h/view?usp=sharing

You'll notice that from a few pixels above the bottom border to the bottom of said border, the tab isn't actually clickable. We should make it so that that region is also clickable.

Reply:
👍